src/site/antora/modules/ROOT/pages/manual/appenders.adoc

```diff
@@ -182,168 +209,6 @@ See xref:manual/configuration.adoc#lazy-property-substitution[runtime property s
 Log4j bundles several predefined appenders to assist in several common deployment use cases.
 Following sections explain all these in detail.
 
-[id=AsyncAppender]
-=== [[asyncappender]] AsyncAppender
```

Comment:
We remove several appenders from the collection and only provide access to them in `nav.adoc`. IMHO, this incoherent navigation style makes it difficult for users to locate the removed appenders. Can't we instead keep them in the list of collections, but provide a link instead? (This is what we have done in `layouts.adoc` for PL and JTL too.)

Reply:
Should be fixed in https://github.com/apache/logging-log4j2/pull/2772/commits/0494163acaf10a1efa76e00ece04a24b6c7acc08